pages/c/_cluster/apps/install.vue

```diff
@@ -921,354 +762,300 @@ export default {
       return { errors, input: out };
     },
 
-    tabChanged({ tab }) {
+    tabChanged() {
       window.scrollTop = 0;
+    },
 
-      if ( tab.name === 'values-yaml' ) {
-        this.$nextTick(() => {
-          if ( this.$refs.yaml ) {
-            this.$refs.yaml.refresh();
-            this.$refs.yaml.focus();
-          }
-        });
-      }
+    updateStepOneReady() {
+      const okRequires = !this.requires.length;
+      const okWarnings = !this.warnings.length || this.ignoreWarning;
+      const okChart = !!this.chart;
+
+      this.steps[0].ready = okRequires && okWarnings && okChart;
     },
 
     getOptionLabel(opt) {
       return opt?.chartDisplayName;
     },
+
   },
 };
 </script>
 
 <template>
   <Loading v-if="$fetchState.pending" />
-
-  <form v-else>
-    <h1 v-if="existing">
-      <t k="catalog.install.header.upgrade" :name="existing.nameDisplay" />
-    </h1>
-    <h1 v-else-if="chart">
-      <t k="catalog.install.header.install" :name="chart.chartDisplayName" />
-    </h1>
-    <h1 v-else>
-      <t k="catalog.install.header.installGeneric" />
-    </h1>
-
-    <div v-if="chart" class="chart-info mb-20">
-      <div class="logo-container">
+  <div v-else class="install-steps">
+    <Wizard
+      v-if="value"
+      :steps="steps"
+      :errors="errors"
+      :edit-first-step="true"
+      :banner-title="stepperName"
+      :banner-title-subtext="stepperSubtext"
+      :finish-mode="action"
+      class="wizard"
+      @cancel="cancel"
+      @finish="finish"
+    >
+      <template #bannerTitleImage>
         <div class="logo-bg">
-          <LazyImage :src="chart.icon" class="logo" />
+          <LazyImage :src="chart ? chart.icon : ''" class="logo" />
         </div>
-      </div>
-      <div class="description">
-        <p>
-          {{ chart.description }}
-        </p>
-      </div>
-    </div>
+      </template>
+      <template #basics>
+        <div class="step__basic">
+          <p v-if="step1Description" class="row mb-10">
+            {{ step1Description }}
+          </p>
+          <div v-if="requires.length || warnings.length" class="mb-30">
+            <Banner v-for="msg in requires" :key="msg" color="error">
+              <span v-html="msg" />
+            </Banner>
+
+            <Banner v-for="msg in warnings" :key="msg" color="warning">
+              <span v-html="msg" />
+            </Banner>
+
+            <Checkbox v-if="warnings.length" v-model="ignoreWarning" label-key="catalog.install.action.ignoreWarning" />
```

Comment:
There's a few different ways in to the stepper (chart install, app details/list, cluster tools) so there's good enough reason for the warnings to be there in the stepper. We could remove it from the chart details page, though it could be handy when flicking to different versions which might trigger/remove warnings. In terms of the check box this was added to mimic the old flow where the warnings had to be acknowledged before starting the process. 

If you'd still like the chart details warnings and/or the stepper warning checkbox removed let me know.

Pluralise has been sorted, good spot.

Reply:
I meant that I don't think the "I agree this may be broken" checkbox before you can continue is necessary.. having the banner in both places and you see one or both depending on how you got there seems like adequate warning.

Reply:
Cool, checkbox removed